plugin.video.uwc/resources/lib/utils.py

```diff
@@ -824,6 +825,16 @@ def searchDir(url, mode, page=None):
         pass
     xbmcplugin.endOfDirectory(addon_handle)
 
+@url_dispatcher.register('920', ['url', 'channel'], ['keyword'])
```

Comment:
@rexdildo 
It crashes on my Windows, Get an error here:
```
File "C:\Users\Test\AppData\Roaming\Kodi\addons\plugin.video.uwc\resources\lib\sites\hentaidude.py", line 25, in <module>
                                                @utils.url_dispatcher.register('920')
                                              File "C:\Users\Test\AppData\Roaming\Kodi\addons\plugin.video.uwc\resources\lib\url_dispatcher.py", line 47, in decorator
                                                raise Exception(message)
                                            Exception: Error: <function Main at 0x000001C75BC6BEB8> already registered as 920
```
Thanks for the contribution.
You can make pull requests to the main branch for Cumination.
Our coders will review it and incorporate it in the next release if they don't have any issues with it.